### PR TITLE
Added application enablement section to  whats new

### PR DESCRIPTION
**Pull request by DawidSAG**

### CODE REVIEW

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 1]
```diff
@@ -5,3 +5,44 @@ layout: bundle
...
+##### Markdown widget
+We are proud to announce the release of Markdown, a new widget designed for displaying files written in markup language. With Markdown, you can now easily present your content in a clear and organized manner. Enhance your display capabilities by incorporating Markdown into your workflow.
+
+![Markdown widget](/images/release-notes/markdown-widget.png)
+
+##### Image widget
+
+We are pleased to announce that our Image widget has been improved with the addition of configuration options for image position and size. Now you have even more control over the display of your images.
+
+![Image widget](/images/release-notes/image-widget.png)
```
Comment (reey) [inline content/release-10-17-0/whatsnew-10-17-0.md:33]: Should we mention the migration to angular? but I guess it is not really of interest for someone reading the overall release notes..

**Review by reey — APPROVED** [review]

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 2]
```diff
@@ -5,3 +5,44 @@ layout: bundle
...
+
+
+
+
+### Application enablement
+
+
+#### User interface redesign
+
+The entire user interface has undergone a complete overhaul, resulting in an improved, more modern and visually appealing design. Our goal was to enhance the user experience and make navigation easier and more intuitive. The redesign features a clean and sleek layout, with clearer typography and more intuitive navigation. Get ready to experience a whole new level of user-friendly design.
```
Comment (janhommes) [inline content/release-10-17-0/whatsnew-10-17-0.md:17]: I would add here information about the improved accessibility.

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 3]
```diff
@@ -5,3 +5,44 @@ layout: bundle
...
+
+##### Image widget
+
+We are pleased to announce that our Image widget has been improved with the addition of configuration options for image position and size. Now you have even more control over the display of your images.
+
+![Image widget](/images/release-notes/image-widget.png)
+
+#### Right drawer
+
+The Right Drawer has undergone a complete redesign with the aim of improving its functionality and usability in daily use. We are confident that these updates will provide a better user experience.
```
[the remark this answers is not in the document]
Comment (DawidSAG) [reply content/release-10-17-0/whatsnew-10-17-0.md:37]: @reey Could you provide some more info?

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 4]
```diff
@@ -5,3 +5,44 @@ layout: bundle
...
+#### Widgets
+
+##### Markdown widget
+We are proud to announce the release of Markdown, a new widget designed for displaying files written in markup language. With Markdown, you can now easily present your content in a clear and organized manner. Enhance your display capabilities by incorporating Markdown into your workflow.
+
+![Markdown widget](/images/release-notes/markdown-widget.png)
+
+##### Image widget
+
+We are pleased to announce that our Image widget has been improved with the addition of configuration options for image position and size. Now you have even more control over the display of your images.
```
[the remark this answers is not in the document]
Comment (DawidSAG) [reply content/release-10-17-0/whatsnew-10-17-0.md:31]: I'll change it to:
```
The "Image" widget has been improved with the addition of configuration options for image position and size. Now you have even more control over the display of your images.
```

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 5]
```diff
@@ -5,3 +5,44 @@ layout: bundle
...
+
+The entire user interface has undergone a complete overhaul, resulting in an improved, more modern and visually appealing design. Our goal was to enhance the user experience and make navigation easier and more intuitive. The redesign features a clean and sleek layout, with clearer typography and more intuitive navigation. Get ready to experience a whole new level of user-friendly design.
+
+![Application redesign](/images/release-notes/ux-redesign.png)
+
+
+#### Widgets
+
+##### Markdown widget
+We are proud to announce the release of Markdown, a new widget designed for displaying files written in markup language. With Markdown, you can now easily present your content in a clear and organized manner. Enhance your display capabilities by incorporating Markdown into your workflow.
```
[the remark this answers is not in the document]
Comment (DawidSAG) [reply content/release-10-17-0/whatsnew-10-17-0.md:25]: What about:
```
The "Markdown" widget has been added, a new widget designed for displaying files written in markup language. Enhance your display capabilities by incorporating "Markdown" widget into your workflow.
```
?

File: content/release-10-17-0/whatsnew-10-17-0.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (DawidSAG) [reply content/release-10-17-0/whatsnew-10-17-0.md:17]: @BeateRixen we have two new versions from the Delite team with small differences:
```
Delite 2.0 is Software AG's in-house design system aimed at establishing
 a common design language for the company's products, enhancing 
usability and accessibility. It features updated color palettes for UI, 
data visualization, and product illustrations to align with the Software
 AG brand and improve accessibility. The design system also includes 
accessibility improvements for screen readers, keyboards, and 
components.
```
and 
```
Delite is Software AG’s in-house design system that is engaged in creating a shared design language for Software AG’s products with the goal of improving and standardizing usability and accessibility across the board. The Delite 2.0 design system comes with new color palettes for UI, data visualization, and product illustrations, that have been created to improve accessibility and alignment to the new Software AG brand. Additional accessibility updates have been made to screen reader, keyboards, and components.
```

File: content/release-10-17-0/whatsnew-10-17-0.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (DawidSAG) [reply content/release-10-17-0/whatsnew-10-17-0.md:37]: @BeateRixen what about:
```
The right drawer has undergone a comprehensive overhaul with the objective of enhancing its functionality and usability in daily use. We are confident that these updates will result in a superior user experience. The changes made to the right drawer include:

- relocating of all user-related settings to it, 
- improved platform versions section for quicker access to more detailed information about the application versions being used,
- information about the global roles assigned to the user,
- updated layout for improved usability.
```

File: content/release-10-17-0/whatsnew-10-17-0.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-17-0/whatsnew-10-17-0.md:17]: I think a like the first one more, how about you @janhommes? But I would slightly change it: 

Remove "and improve accessibility":
to align with the Software AG brand and improve accessibility. > to align with the Software
 AG brand.

As this is the repeated in the following sentence.

File: content/release-10-17-0/whatsnew-10-17-0.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-17-0/whatsnew-10-17-0.md:25]: I would still add some example, what I can display here.
Enhance your display capabilities by incorporating "Markdown" widget into your workflow. >
By incorporating the "Markdown" widget you can enhance your display capabilities to inform users, for example, on new features.
Is that correct @janhommes ?

File: content/release-10-17-0/whatsnew-10-17-0.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-17-0/whatsnew-10-17-0.md:37]: The right drawer has been redesigned to enhance its functionality and usability and thus to improve the user experience. The changes made to the right drawer include:

- relocation of all user-related settings from the user menu to the right drawer, 
- improved platform versions section for quicker access to more detailed information about the application versions being used,
- information about the global roles assigned to the user,
- updated layout for improved usability.

**Comment by BeateRixen** [comment]
@DawidSAG For the markdown widget, I would actually use an image with meaningful content, i.e. showing the widget in real use.

**Comment by BeateRixen** [comment]
@DawidSAG We should add links to the documentation for all items, which brings me to the question if these have been documented (like the new widgets) @janhommes ?

File: content/release-10-17-0/whatsnew-10-17-0.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (DawidSAG) [reply content/release-10-17-0/whatsnew-10-17-0.md:25]: yes, this is one of the use cases. I'll prepare something Today. @BeateRixen Maybe I could reuse some of the old What's New section in it?

**Comment by DawidSAG** [comment]
> @DawidSAG We should add links to the documentation for all items, which brings me to the question if these have been documented (like the new widgets) @janhommes ?

I need to check, for sure we have missed out on creating markdown widget docs. We will add them quickly.

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@janhommes Can this be merged or do you have more comments? I have added generic links to the widgets collection now, please add the documentation for the widgets there as well.

**Review by janhommes — APPROVED** [review]

**Pull request merged**